### PR TITLE
Fix Elasticsearch and Postgres resources

### PR DESCRIPTION
**Pull request by luisarojas**
### Changed

* Elasticsearch and PostgreSQL Terraform tests
* Elasticsearch and PostgreSQL cost components for disk, memory, and CPU
* Golden file contents to account for cost component changes and recent scraper fix

### Added

* New cost component for flavour
* Parsing of Terraform resource `ibm_database` to pull disk, memory, cpu, members and flavour

### Removed

* All usage-based variables for `ibm_database` resource

### CODE REVIEW

File: internal/resources/ibm/database_elasticsearch.go [hunk 1]
```diff
@@ -1,122 +1,133 @@
...
-	if r.ElasticSearch_Members != nil {
-		m = decimal.NewFromInt(*r.ElasticSearch_Members)
-	} else { // set the default
-		m = decimal.NewFromInt(3)
+
+	return &schema.CostComponent{
+		Name:            fmt.Sprintf("RAM (%s members)", strconv.FormatInt(r.Members, 10)),
+		Unit:            "GB",
+		UnitMultiplier:  decimal.NewFromInt(1),                                         // Final quantity for this cost component will be divided by this amount
+		MonthlyQuantity: decimalPtr(decimal.NewFromInt((r.Memory * r.Members) / 1024)), // Convert to GB
```
Comment (YuChinKoo) [inline internal/resources/ibm/database_elasticsearch.go:61]: will there ever be a case where the value is not divisible by 1024?

Comment (luisarojas) [reply internal/resources/ibm/database_elasticsearch.go:61]: @YuChinKoo nope, the terraform only allows that value to be in increments of 1024

Comment (luisarojas) [reply internal/resources/ibm/database_elasticsearch.go:61]: Let me test it...

Comment (luisarojas) [reply internal/resources/ibm/database_elasticsearch.go:61]: * memory must be >= 1024 and <= 114688 in increments of 128
* disk must be >= 5120 and <= 4194304 in increments of 1024

Comment (luisarojas) [reply internal/resources/ibm/database_elasticsearch.go:61]: Changed it so that it will display a float instead of an int 👍🏼

**Review by YuChinKoo — APPROVED** [review]
lgtm

**Pull request merged**